io.github.face_hh.Napture.json

```diff
@@ -0,0 +1,79 @@
+{
+    "app-id": "io.github.face_hh.Napture",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "napture",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--own-name=org.bussin.napture"
```

Comment:
Why is the busname different than the appid? If the developer does not control the bussin.org then the bus name should be changed to match the appid.

If they do, the id should be `org.bussin.napture`.